knora/dsp_tools.py

```diff
@@ -204,6 +221,12 @@ def program(user_args: list[str]) -> None:
         excel2xml(datafile=args.datafile,
                   shortcode=args.shortcode,
                   default_ontology=args.default_ontology)
+    elif args.action == 'start-api' and not sys.platform.startswith('win'):
```

Comment:
This is probably a better way to check the OS, I guess?

Reply:
not sure how that would behave with WSL... but probably you shouldn't worry about that too much. So this looks good to me